components/Dialogs/EstimateChangeDialog.tsx

```diff
@@ -9,13 +10,27 @@ import { editStory } from '../../redux/actions/stories.actions';
 import { Story } from '../../redux/types';
 import EstimatePicker from '../EstimatePicker';
 
+const ESTIMATE_REMOVAL_SUPPORTED_STAES = ['unstarted', 'unscheduled'];
```

Comment:
this name is a little confusing to me, how about

`ESTIMATE_NOT_REQUIRED_STATES`